lstchain/mc/sensitivity.py

```diff
@@ -178,15 +182,8 @@ def calculate_sensitivity_lima(n_on_events, n_background, alpha, n_bins_energy,
     """
 
     
-    stat = WStatCountsStatistic(
-        n_on=n_on_events,  
-        n_off=n_background,
-        alpha=alpha
-        )
-
-
-    n_excesses_5sigma = stat.excess_matching_significance(5)
-
+    n_excesses_5sigma = excess_matching_significance(n_on_events, n_background, alpha, 5)
```

Comment:
Actually, you could replace this function completely by just using

```
sensitivity = relative_sensitivity(n_on, n_off, alpha)
n_excess = sensitivity * (n_on - alpha * n_off)
return n_excess, sensitivity
```

as pyirf also takes care of the condition checks. The loops are not needed (even now they are not needed and could be replaced by simple masking).

Reply:
ok, thanks for the suggestion, we can think on improvements in the future, but I think now it is important that the `gammapy` issue, which was the original reason for this PR, gets solved